roles/metallb_setup/tasks/pre-requisites.yml

```diff
@@ -80,6 +80,13 @@
   when:
     - mlb_bgp_peers is defined
 
+- name: "Validate values of mlb_wait_* are positive integers"
+  ansible.builtin.assert:
+    that:
+      - mlb_wait_delay | int > 0
+      - mlb_wait_retries | int > 0
+    fail_msg: "The parameters mlb_wait_delay and mlb_wait_retries must both be integers"
+
```

Comment:
Could you move the validation before its usage around line 51 or before, thanks!

Reply:
I had forgotten about its use in the pre-requisites task list. Good catch!